app/data/0_star_wars.json

```diff
@@ -127,6 +127,11 @@
                                 }
                             }
 
+                        },
+                        {
+                            "goto": {
+                                "id": "96682325-47ab-41e4-a56e-8315a19ffe2a"
+                            }
```

Comment:
we need to specify, the way the navigator works if routing_rules are defined then 1 of them needs to match or default